### PR TITLE
Fix 'edit specific price' modal behavior bugs and display

### LINKED ISSUE #10175: Edit a specific price issues
**Describe the bug**

Can we improve the display of the fields below ? 
It's not aligned and the discount and tax fields are very small, not easy to see what are inside

capture d ecran_207

When I try to change the initial product price with a comma, I have the error below

capture d ecran_208

When I try to save a specific price with a comma, the numbers after it are not saved.
For exemple, if I try to save 12,45 it will save 12

When I try to delete the customer and add a new one, I can't save because I don't see the button

capture d ecran_209

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fixes 'edit specific price' modal behavior bugs and bad display
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #10175
| How to test?  | The bugs list is in the Issue. No behavior described in the Issue should remain with this PR, everything should be fixed.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10202)

### CODE REVIEW

**Comment by matks** [comment]
Fixes #10175

File: admin-dev/themes/new-theme/js/pages/catalog/product/specific-price-form-handler.js [hunk 1]
```diff
@@ -214,6 +217,16 @@ class SpecificPriceFormHandler {
     });
   }
 
+  /**
+   * @see https://vijayasankarn.wordpress.com/2017/02/24/quick-fix-scrolling-and-focus-when-multiple-bootstrap-modals-are-open/
+   */
+  configureMultipleModalsBehavior() {
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/js/pages/catalog/product/specific-price-form-handler.js:221]: In which case you would have multiple modals?

Comment (matks) [reply admin-dev/themes/new-theme/js/pages/catalog/product/specific-price-form-handler.js:221]: Specific price form has a field "customer" which allows to type a customer name, performs an ajax search and allows to select it. If you want to clear this field, the "remove" button opens another modal.

**Review by eternoendless — DISMISSED** [review]

**Comment by marionf** [comment]
@matks 

Just a last thing: When I edit a specific price in %, I see the tax incl/tax excl drop down
It shouldn't be displayed if % is selected

capture d ecran_240

Otherwise, everything is good :+1:

**Comment by matks** [comment]
Bug found by marionf fixed by https://github.com/PrestaShop/PrestaShop/pull/10202/commits/7f887b78cd9348ef340f0fcf307462001f0c7f52
An additionnal modal bug fixed by https://github.com/PrestaShop/PrestaShop/pull/10202/commits/50d4c57a0cda60a92fd988e7b05452ce48dfb078

Need review for these 2 commits :)

File: admin-dev/themes/new-theme/js/pages/catalog/product/specific-price-form-handler.js [hunk 2]
```diff
@@ -154,6 +154,7 @@ class SpecificPriceFormHandler {
     var selectorPrefix = this.getPrefixSelector(usePrefixForCreate);
 
     $('#form_modal_cancel').click(() => this.closeEditPriceModalAndRemoveForm());
+    $('#form_modal_close').click(() => this.closeEditPriceModalAndRemoveForm());
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/pages/catalog/product/specific-price-form-handler.js:157]: You also can do this, but not mandatory.
`$('#form_modal_cancel, #form_modal_close').on('click', (() =>...`

**Review by PierreRambaud — DISMISSED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @matks

**Pull request merged**